### PR TITLE
Updated README with more clear instructions

### PR DESCRIPTION
**Pull request by kevando**
I've re-ordered what you already had here and added some steps to help people that may not have this set up. 

Feedback is welcome and I still want to go through this again myself and make sure I didnt miss anything

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,51 +1,100 @@
...
 
-Right now it only has two proposal and it does not allow any aditional proposal so you can vote either for
+This is an example smart contract on the [NEAR protocol](https://near.org/developers/). This example keeps track of votes for a proposal that you can deploy on the NEAR blockchain. These are the steps to deploy to the testnet.
 
-- NFT MAKER APP
-- HomePage
+## Instructions
 
-### Contract Name
+The example is intended for developers and some level of techincal expertise is assumed. If you don't know what terminal is, this probably isn't for you.
```
Comment (fioritonicolas) [inline README.md]: I would remove this part _If you don't know what terminal is, this probably isn't for you._  just to be more friendly :)

File: README.md [hunk 2]
```diff
@@ -1,51 +1,100 @@
...
 
-votingnico.testnet
 
-Deployed https://explorer.testnet.near.org/transactions/HpyWnTM4KUnt45MoQiAgwco2pMSQqDmoSn2CwtU6N4mU
+### Prerequisites
```
Comment (fioritonicolas) [inline README.md:5]: Nice, prerequisites were needed!

File: README.md [hunk 3]
```diff
@@ -1,51 +1,100 @@
...
+
+1. NFT MAKER APP
+2. HomePage
+
+Here's how you vote for NFT Maker
+
+```
+near call mytestaccount.testnet vote '{"proposal_name":"NFT MAKER APP"}' --accountId mytestaccount.testnet
+```
+
```
Comment (fioritonicolas) [inline README.md:81]: what about if we explain these params. It took me a while to understand it, and is not that clear on the near docs either.

Something like:

near call receives first the contract name, then one of the public functions in the contract, if this function accepts parameters next to the function name should be passed, and last the signing account which is calling this contract, this account is has to be the same that you logged in in the previous step.

**Review by fioritonicolas — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (kevando) [reply README.md:81]: Nice explanation1

File: README.md (hunk 1 above)
Comment (kevando) [reply README.md]: haha yes good idea to be friendly the first paragraph :)

i removed this line

**Pull request merged**